src/main/charts/confluence/templates/synchrony-start-script.yaml

```diff
@@ -11,7 +11,9 @@ data:
     #!/usr/bin/env bash
 
     java \
+       -Xss{{ .Values.synchrony.resources.jvm.stackSize }} -Xmx{{ .Values.synchrony.resources.jvm.maxHeap }} \
+       -XX:ActiveProcessorCount={{ .Values.synchrony.resources.container.requests.cpu }} \
```

Comment:
@errcode1202 this could be a potential way how to automate the CPU requests & jvm args. Did we test with `m` units? E.g. `-XX:ActiveProcessorCount=250m`?

Reply:
Yup millicores were tested. From what I remember it will round up/down to the nearest whole cpu. My question here is why does only `250m` (0.25) of a cpu for synchrony? Why not just give it a whole core?